pos_cash_move_reason/models/pos_move_reason.py

```diff
@@ -12,9 +12,7 @@ class PosMoveReason(models.Model):
     @api.model
     def _default_journal_ids(self):
         AccountJournal = self.env["account.journal"]
-        journals = AccountJournal.search(
-            [("journal_user", "=", True), ("type", "=", "cash")]
-        )
+        journals = AccountJournal.search([("type", "=", "cash")])
```

Comment:
why removing the ``journal_user`` constraint ?

Reply:
This field doesn't exist in v`13.0`